components/SurveyModal.js

```diff
@@ -0,0 +1,566 @@
+  updateForm: PropTypes.func.isRequired,
+  stepError: PropTypes.string,
+};
+
+function Step4(props) {
+  const { skipStep, validateStep, updateForm, stepError } = props;
+  const requiredFields = ['firstName', 'lastName', 'email'];
+  return (
+    <form
+      id="mailchimp-form"
+      name="mailchimp-form"
+      className="tji-modal__form"
+      // action="https://texasjusticeinitiative.us18.list-manage.com/subscribe/post?u=fd262cb4a5fc0bafb38da2e22&amp;id=2663621fac"
+      // method="post"
+      // target="_blank"
+    >
+      <h2 className="tji-modal__title">I'm interested in TJI updates...</h2>
+      <p className="tji-modal__description">
+        Thank you for your interest in TJI! For regular updates on TJI’s data in use, our latest data offerings and the
+        most recent revelations, sign up for our newsletter, a short read that hits inboxes monthly. We promise not to
+        sell your email address.
+      </p>
+      <fieldset>
+        <input
+          style={{ marginBottom: '0.5em' }}
+          type="text"
+          placeholder="First name"
+          name="FNAME"
+          onChange={event => updateForm(event, 'firstName')}
+        />
+        <input
+          style={{ marginBottom: '0.5em' }}
+          type="text"
+          placeholder="Last Name"
+          name="LNAME"
+          onChange={event => updateForm(event, 'lastName')}
+        />
+        <input
+          style={{ marginBottom: '0.5em' }}
+          type="email"
+          placeholder="Email"
+          name="EMAIL"
+          onChange={event => updateForm(event, 'email')}
+        />
+      </fieldset>
+      <div className="tji-modal__actions">
+        <button type="button" onClick={() => skipStep()} className="btn--simple">
+          No Thanks
+        </button>
+        <button
+          type="button"
+          className="btn btn--primary"
+          onClick={() => validateStep(requiredFields, newsletterCallback)}
+        >
+          Continue
+        </button>
+      </div>
+      {stepError !== '' && <p className="tji-modal__form__error">{stepError}</p>}
+    </form>
+  );
+}
+
+Step4.propTypes = {
+  skipStep: PropTypes.func.isRequired,
+  validateStep: PropTypes.func.isRequired,
+  updateForm: PropTypes.func.isRequired,
+  stepError: PropTypes.string,
+};
+
+function Step5(props) {
+  const { cancelForm } = props;
+  return (
+    <form className="tji-modal__form">
+      <h2 className="tji-modal__title">Before you go...</h2>
+      <p className="tji-modal__description">
+        TJI provides its data to all users for free. Please consider making a contribution so we can continue to be a
+        free resource.
+      </p>
+      <div className="tji-modal__actions">
+        <button type="button" onClick={() => cancelForm()} className="btn--simple">
+          Not Now
+        </button>
+        <button type="button" className="btn btn--primary" onClick={() => (window.location.href = 'donate')}>
```

Comment:
When I clicked the "donate" button, I got a 404 because I was sent to `/data/donate`. I think this should fix it.

```suggestion
        <button type="button" className="btn btn--primary" onClick={() => (window.location.href = '/donate')}>
```

Reply:
What do you think about opening the download page in a new tab? I'm thinking of the scenario where somebody sets up a bunch of specific filters and doesn't want to lose them.

Reply:
I think you meant opening the donate page in a new tab. I like that idea, and implemented it in the latest update.

Reply:
> I think you meant opening the donate page in a new tab.

Whoops! Yep, totally.

Looks great! 👍